@xen-orchestra/lite/src/stories/ui-button.story.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <ComponentStory
+    :params="[
+      prop('type').enum('submit', 'button', 'reset').default('button'),
+      prop('busy').bool().widget(),
+      prop('disabled').bool().widget(),
+      iconProp(),
+      colorProp(),
+      prop('outlined').bool().widget(),
+      prop('transparent').bool().widget(),
+      prop('active').bool().widget(),
+      setting('label').preset('Click me').widget(),
```

Comment:
The `event` helper is meant to document custom events (defined with `defineEmits`).

Here `@click` would lead to a native event, and I don't think it should be documented (we don't document `@dblclick`, `@focus`, `@blur`...)